lisa/sut_orchestrator/azure/features.py

```diff
@@ -1913,10 +1912,13 @@ def enabled(self) -> bool:
 class ArchitectureSettings(schema.FeatureSettings):
     type: str = "Architecture"
     # Architecture in hyper-v
-    arch: str = field(
-        default="x64",
+    arch: Optional[Union[search_space.SetSpace[str], str]] = field(  # type:ignore
```

Comment:
Keep it as string, and assert req equals cap in `_call_requirement_method`. The reason is that the arch is exclusive each other on both image and vm size. It's impossible to support both in one of them.